astroquery/utils/tap/tests/test_tap.py

```diff
@@ -878,18 +876,18 @@ def __find_table(schemaName, tableName, tables):
     pytest.fail(f"Table '{qualified_name}' not found")
 
 
-def __find_column(column_name, columns):
-    for c in columns:
-        if c.name == column_name:
+def __find_column(columnName, columns):
+    for c in (columns):
+        if c.name == columnName:
```

Comment:
I suppose this got reverted to the previous camelcase in the rebase gone wrong, in the newly rebased version this is being fixed now.